curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/pig-latin.english.md

```diff
@@ -11,6 +11,7 @@ forumTopicId: 16039
 Translate the provided string to pig latin.
 <a href="http://en.wikipedia.org/wiki/Pig_Latin" target="_blank">Pig Latin</a> takes the first consonant (or consonant cluster) of an English word, moves it to the end of the word and suffixes an "ay".
 If a word begins with a vowel you just add "way" to the end.
+If a word begins with consonants, move them to the end followed by "ay".
```

Comment:
The next line already explains what to do if the word does not contain a vowel.  It says to `just add "ay" to the end`.

Also, your addition is not accurate, because just because the word starts with consonants, does not mean it does not have vowels after those consonants.

Reply:
It says if it contains no vowels, which is very specific. The words you convert contain vowels, and my statement is after it already tells you to move consonants to the end of the word, however that bit is tucked away and not prominent like the other two statements.

Reply:
The definition of Pig Latin already explains what should happen with the first consonant or consonant cluster:
> Pig Latin</a> takes the first consonant (or consonant cluster) of an English word, moves it to the end of the word and suffixes an "ay".

Reply:
If it explained it well I wouldn't be here with a suggestion. On the description it clearly displays the rules separate from the first sentence. I do not see why you are against clarifying a prompt to avoid confusion.

I find it pointless to argue about opinion on how to display something, I'll let whomever has the ability to review it decide, however I did waste time trying to complete this incorrectly because of how it is presented.